### PR TITLE
feat: removed conditional network-related choice for the managing modal

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The item in the overflow menu is now the same for all the networks.

# Changes

Removed IF statement from the tokens menu.

# To-Do

- [ ] Remove deprecated components

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/TokensMenu.svelte [hunk 1]
```diff
@@ -28,16 +26,10 @@
...
+			<Hr />
+		</div>
 
-			{#if $networkICP}
-				<IcManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
-			{:else if $networkEthereum}
-				<ManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
-			{/if}
-		{/if}
+		<IcManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/TokensMenu.svelte:33]: This component will be renamed (to remove the Ic prefix) and moved in another PR I guess?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/tokens/TokensMenu.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/tokens/TokensMenu.svelte:33]: correct!

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/tokens/TokensMenu.svelte:33]: created PR #1523
